modules/badge/react/lib/CountBadge.tsx

```diff
@@ -53,9 +54,9 @@ const Container = styled('span')<CountBadgeProps>(
 );
 
 const CountBadge = (props: CountBadgeProps) => {
-  const {variant = 'default', count = 0, ...elemProps} = props;
+  const {variant = 'default', count = 0, limit = 1000, ...elemProps} = props;
 
-  const formattedCount = count < 1000 ? `${count}` : '999+';
+  const formattedCount = count < limit ? `${count}` : `${limit - 1}+`;
```

Comment:
Note: The math here won't work if numbers are negative. haha

We could validate the limit here "needs to be greater than 0" or something like that, but I kind of want to keep the badge as dumb as possible.